public/javascripts/gtresolution.js

```diff
@@ -86,7 +88,21 @@ $(document).ready(function () {
                 status: "Ground_Truth",
                 zIndex: 200
             }
-        }, //filter labels are dealth with in low disgareement round
+        },
+        // When only one person placed GT labels (dealt with in modified low disagreement round)
```

Comment:
"within"? :)

Reply:
I believe "with in" is correct here :grin: 

We are dealing with, not dealing!